contracts/MemeFactory.sol

```diff
@@ -529,8 +529,8 @@ abstract contract MemeFactory {
         // Get meme summon info
         MemeSummon memory memeSummon = memeSummons[memeNonce];
 
-        // Check if the meme has been summoned
-        require(memeSummon.unleashTime > 0, "Meme not unleashed");
+        // Check if the meme has been unleashed
+        require(memeSummon.unleashTime > 1, "Meme not unleashed");
```

Comment:
Ugly - but gets the job done. Otherwise this is also a major bug for campaign launches...

Reply:
Initially I thought this was a bug. But because memeToken is not known until unleash happened, there's no way for this to be reached even though `memeSummon.unleashTime > 0` holds for launchNone